fix: Add pre-IA history access instructions (#4630)
newrelic/docs-website: src/content/docs/style-guide/writer-workflow/github-intro.mdx

**Option 3: Git blame**

Follow [Github's documentation](https://docs.github.com/en/github/managing-files-in-a-repository/managing-files-on-github/tracking-changes-in-a-file).

<Callout variant="important">

> We had a large site restructure in October 2021 which lost most of file history for our docs. You can find an archived version of our site pre-rework in the [pre-IA-2021](https://github.com/newrelic/docs-website/tree/pre-IA-2021). You can navigate our the pre-rework version of our repo, including any now lost file history.

Reviewer: ```suggestion
We had a large site restructure in October 2021 which lost most of the file history for our docs. You can find an archived version of our site pre-rework in the [pre-IA-2021](https://github.com/newrelic/docs-website/tree/pre-IA-2021) branch. By navigating the pre-rework version of our repo, you can find file history and more.
```

Alternatively, you can use the following command in your terminal:
```
git log --follow "**/file_name_here.mdx"

> ```

Reviewer: It would be great to see a sample (like before - you can even use that file name).

> This will output the commit history of that file. By default, it only shows the first few commits. You can scroll by pressing enter multiple times.

Reviewer: ```suggestion
This will output the commit history of that file. By default, it only shows the first few commits. You can scroll by pressing Return multiple times.
```
